SingularityUI/webpack.config.js

```diff
@@ -68,7 +70,7 @@ module.exports = {
       }
     }),
     new webpack.DefinePlugin({
-      'process.env.NODE_ENV': '"production"'
+      'process.env.NODE_ENV': JSON.stringify('development')
```

Comment:
Oh, right. I forgot that I put this in there.

We should figure out if there's a way to override the default behavior and not do this.

Minified react (which is used when you use `production`) doesn't have good error messages. I wanted to see what the issue was with something, and they were being suppressed. By the time this patch gets merged I'd like to see this behavior default to production but use development if specified on the command line.